cylc/flow/cfgspec/globalcfg.py

```diff
@@ -99,6 +99,53 @@
         'sort order': [VDR.V_STRING, 'definition', 'alphanumeric'],
     },
 
+    # job platforms
+    'job platforms': {
+        '__MANY__': {
+            'batch system': [VDR.V_STRING, 'background'],
+            'batch submit command template': [VDR.V_STRING],
+            'shell': [VDR.V_STRING, '/bin/bash'],
```

Comment:
Added here in place of the current task runtime setting - there should be no need to be set this for an individual task.